### ISSUE TITLE
existing object is not controlled by any of UID

### ISSUE THREAD

**Issue by zengzezheng** [kubevela/kubevela#2606]
My application is modifying environment variables and expanding or reducing capacity ,  `describe app`   There was a mistake like that,

![image](https://user-images.githubusercontent.com/72589839/139821964-c5c73cb0-4efc-482a-a2db-cfad0520aa53.png)

After this error occurs,I'm using a new `yaml` , this  `yaml`  was modified  the environment variable  ,    after `apply`  ,Didn't update my pod 

I  always modify   value of  `startFlag`  or  `replicas`,  my yaml is

```
apiVersion: core.oam.dev/v1beta1
kind: Application
metadata:
  name: paas-test
  namespace: tenant-paas-test-liming
  labels:
    ns: tenant-paas-test-liming
spec:
  components:
  - name: paas-test-nacos-secret
    type: raw
    properties:
      apiVersion: v1
      kind: Secret
      metadata:
        name: paas-test-nacos-secret
        namespace: tenant-paas-test-liming
      type: generic
      data:
        MYSQL_SERVICE_HOST: cGFhcy10ZXN0LXBhYXMtbXlzcWwudGVuYW50LXBhYXMtdGVzdC1saW1pbmcuc3ZjLmNsdXN0ZXIubG9jYWw=
  - name: paas-test-nacos
    type: webservice
    properties:
      env:
      - name: startFlag
        value: 568fba81-4a40-41d0-8e57-0148991807f9
      - name: PREFER_HOST_MODE
        value: hostname
      - name: MODE
        value: standalone
      - name: SPRING_DATASOURCE_PLATFORM
        value: mysql 
      - name: MYSQL_SERVICE_DB_NAME
        value: nacos_devtest
      - name: MYSQL_SERVICE_USER
        value: root
      - name: MYSQL_SERVICE_PASSWORD
        value: root
      - name: MYSQL_SERVICE_PORT
        value: "3306"
      image: nacos/nacos-server:2.0.3
      port: 8848
      cpu: 300m
      memory: 2000M
    dependsOn:
    - paas-test-paas-mysql
    traits:
    - type: ingress
      properties:
        http:
          /: 8848
        domain: paas.nacos.com
    - type: scaler
      properties:
        replicas: 1
    - type: service-binding
      properties:
        envMappings:
          MYSQL_SERVICE_HOST:
            secret: paas-test-nacos-secret
  - name: paas-test-paas-gateway-secret
    type: raw
    properties:
      apiVersion: v1
      kind: Secret
      metadata:
        name: paas-test-paas-gateway-secret
        namespace: tenant-paas-test-liming
      type: generic
      data:
        nacos-addr: cGFhcy10ZXN0LW5hY29zLnRlbmFudC1wYWFzLXRlc3QtbGltaW5nLnN2Yy5jbHVzdGVyLmxvY2Fs
  - name: paas-test-paas-gateway
    type: webservice
    properties:
      env:
      - name: startFlag
        value: 5c01e89b-3fc5-48c6-97e8-aa14f7a1242f
      - name: namespace
        value: kubewanwei-develop
      image: domain/kubetest/test-gateway-1025
      port: 19090
      cpu: 300m
      memory: 500M
    dependsOn:
    - paas-test-nacos
    traits:
    - type: ingress
      properties:
        http:
          /: 19090
        domain: gateway.paas.com
    - type: scaler
      properties:
        replicas: 1
    - type: service-binding
      properties:
        envMappings:
          nacos-addr:
            secret: paas-test-paas-gateway-secret
  - name: paas-test-paas-mysql
    type: webservice
    properties:
      env:
      - name: startFlag
        value: 4738c65f-d4e4-4c5e-a35b-8d91be169d09
      - name: MYSQL_ROOT_PASSWORD
        value: root
      image: mysql:5.7
      port: 3306
      cpu: 300m
      memory: 1000M
    traits:
    - type: tcpservice
      properties:
        http:
          /: 3306
    - type: scaler
      properties:
        replicas: 1
    - type: pvc
      properties:
        claimName: mysqldata
        accessModes:
        - ReadWriteOnce
        storageClassName: nfs-fbsz
        resources:
          requests:
            storage: 2Gi
        volumesToMount:
        - mountPath: /var/lib/mysql
          name: mysqldata
  - name: paas-test-front-web
    type: webservice
    properties:
      env:
      - name: startFlag
        value: decef7ce-e18f-4ae9-b878-b7df2a0c4d5d
      - name: mysql-port
        value: "3306"
      - name: mysql-user
        value: root
      - name: mysql-password
        value: root
      image: domain/kubetest/front-admin:1027
      port: 80
      cpu: 300m
      memory: 500M
    traits:
    - type: ingress
      properties:
        http:
          /: 80
        domain: front.paas.com
    - type: scaler
      properties:
        replicas: 1
  - name: paas-test-tenant-secret
    type: raw
    properties:
      apiVersion: v1
      kind: Secret
      metadata:
        name: paas-test-tenant-secret
        namespace: tenant-paas-test-liming
      type: generic
      data:
        nacos-addr: cGFhcy10ZXN0LW5hY29zLnRlbmFudC1wYWFzLXRlc3QtbGltaW5nLnN2Yy5jbHVzdGVyLmxvY2Fs
        mysql-addr: cGFhcy10ZXN0LXBhYXMtbXlzcWwudGVuYW50LXBhYXMtdGVzdC1saW1pbmcuc3ZjLmNsdXN0ZXIubG9jYWw=
  - name: paas-test-tenant
    type: webservice
    properties:
      env:
      - name: startFlag
        value: 72a549fb-5089-4d3e-9082-733bba9c6c8f
      - name: namespace
        value: kubewanwei-develop
      image: domain/kubetest/tenant:test
      port: 8084
      cpu: 300m
      memory: 500M
    dependsOn:
    - paas-test-nacos
    - paas-test-paas-mysql
    traits:
    - type: innerservice
      properties:
        http:
          /: 8084
    - type: scaler
      properties:
        replicas: 1
    - type: service-binding
      properties:
        envMappings:
          nacos-addr:
            secret: paas-test-tenant-secret
          mysql-addr:
            secret: paas-test-tenant-secret
  - name: paas-test-paas-application-secret
    type: raw
    properties:
      apiVersion: v1
      kind: Secret
      metadata:
        name: paas-test-paas-application-secret
        namespace: tenant-paas-test-liming
      type: generic
      data:
        nacos-addr: cGFhcy10ZXN0LW5hY29zLnRlbmFudC1wYWFzLXRlc3QtbGltaW5nLnN2Yy5jbHVzdGVyLmxvY2Fs
        mysql-port: MzMwNg==
        mysql-user: cm9vdA==
        mysql-host: cGFhcy10ZXN0LXBhYXMtbXlzcWwudGVuYW50LXBhYXMtdGVzdC1saW1pbmcuc3ZjLmNsdXN0ZXIubG9jYWw=
  - name: paas-test-paas-application
    type: webservice
    properties:
      env:
      - name: startFlag
        value: 44ce7ba8-9d5f-4306-bcbb-bf02359b5599
      - name: namespace
        value: kubewanwei-develop
      - name: mysql-password
        value: root
      image: domain/kubetest/application-web:test
      port: 8086
      cpu: 500m
      memory: 500M
    dependsOn:
    - paas-test-nacos
    - paas-test-tenant
    - paas-test-paas-mysql
    traits:
    - type: innerservice
      properties:
        http:
          /: 8086
    - type: scaler
      properties:
        replicas: 1
    - type: service-binding
      properties:
        envMappings:
          nacos-addr:
            secret: paas-test-paas-application-secret
          mysql-port:
            secret: paas-test-paas-application-secret
          mysql-user:
            secret: paas-test-paas-application-secret
          mysql-host:
            secret: paas-test-paas-application-secret
  - name: paas-test-paas-cluster-secret
    type: raw
    properties:
      apiVersion: v1
      kind: Secret
      metadata:
        name: paas-test-paas-cluster-secret
        namespace: tenant-paas-test-liming
      type: generic
      data:
        nacos-addr: cGFhcy10ZXN0LW5hY29zLnRlbmFudC1wYWFzLXRlc3QtbGltaW5nLnN2Yy5jbHVzdGVyLmxvY2Fs
        mysql-port: MzMwNg==
        mysql-user: cm9vdA==
        mysql-host: cGFhcy10ZXN0LXBhYXMtbXlzcWwudGVuYW50LXBhYXMtdGVzdC1saW1pbmcuc3ZjLmNsdXN0ZXIubG9jYWw=
  - name: paas-test-paas-cluster
    type: webservice
    properties:
      env:
      - name: startFlag
        value: 61014fee-9ec7-4d07-85f1-eef04a16882f
      - name: mysql-password
        value: root
      - name: namespace
        value: kubewanwei-develop
      image: domain/kubetest/cluster:test
      port: 8081
      cpu: 500m
      memory: 500M
    dependsOn:
    - paas-test-nacos
    - paas-test-paas-mysql
    traits:
    - type: innerservice
      properties:
        http:
          /: 8081
    - type: scaler
      properties:
        replicas: 1
    - type: service-binding
      properties:
        envMappings:
          nacos-addr:
            secret: paas-test-paas-cluster-secret
          mysql-port:
            secret: paas-test-paas-cluster-secret
          mysql-user:
            secret: paas-test-paas-cluster-secret
          mysql-host:
            secret: paas-test-paas-cluster-secret
  - name: paas-test-paas-kubernetes-secret
    type: raw
    properties:
      apiVersion: v1
      kind: Secret
      metadata:
        name: paas-test-paas-kubernetes-secret
        namespace: tenant-paas-test-liming
      type: generic
      data:
        nacos-addr: cGFhcy10ZXN0LW5hY29zLnRlbmFudC1wYWFzLXRlc3QtbGltaW5nLnN2Yy5jbHVzdGVyLmxvY2Fs
        mysql-port: MzMwNg==
        mysql-user: cm9vdA==
        mysql-host: cGFhcy10ZXN0LXBhYXMtbXlzcWwudGVuYW50LXBhYXMtdGVzdC1saW1pbmcuc3ZjLmNsdXN0ZXIubG9jYWw=
  - name: paas-test-paas-kubernetes
    type: webservice
    properties:
      env:
      - name: startFlag
        value: 960e50cf-0c52-421a-ae5a-1786568f36b6
      - name: namespace
        value: kubewanwei-develop
      image: domain/kubetest/kubernetes-web:test
      port: 8088
      cpu: 500m
      memory: 500M
    dependsOn:
    - paas-test-paas-cluster
    - paas-test-nacos
    - paas-test-paas-mysql
    traits:
    - type: innerservice
      properties:
        http:
          /: 8088
    - type: scaler
      properties:
        replicas: 1
    - type: service-binding
      properties:
        envMappings:
          nacos-addr:
            secret: paas-test-paas-kubernetes-secret
          mysql-port:
            secret: paas-test-paas-kubernetes-secret
          mysql-user:
            secret: paas-test-paas-kubernetes-secret
          mysql-host:
            secret: paas-test-paas-kubernetes-secret
  policies:
  - name: health-policy
    type: health
    properties:
      probeTimeout: 10
      probeInterval: 10

```

**Comment by Somefive** [comment]
Try to upgrade KubeVela to latest version (1.2.0-beta.3), a newer resource management mechanism is designed to handle such issues.

**Comment by wonderflow** [comment]
closing, it's fixed in v1.2 release
